.pipelines/e2e-windows.yaml

```diff
@@ -72,4 +80,32 @@ stages:
       windowsVMSize: Standard_DS2_v2
       windowsDistro: aks-windows-2022-containerd-gen2
       windowsNodeImageVersion: AKSWindows-2022-containerd-gen2-test-2023.02.07
-      storageAccount: akswinstore2022gen2
\ No newline at end of file
+      storageAccount: akswinstore2022gen2
+
+- stage: test_2022_containerd_gen2_gpu_grid
+  dependsOn: []
+  condition: eq('${{ parameters.test2022containerdgen2gpugrid }}', True)
+  jobs:
+  - template: ./templates/e2e-windows-template.yaml
+    parameters:
+      windowsImage: 2022-containerd-gen2
+      windowsOSSKU: Windows2022
+      windowsVMSize: Standard_NV6ads_A10_v5
+      windowsDistro: aks-windows-2022-containerd-gen2
+      windowsNodeImageVersion: AKSWindows-2022-containerd-gen2-test-2023.02.07
```

Comment:
could we use latest instead of a specified version to avoid updating it from time to time?

Reply:
I am following the existing pattern for non-GPU Windows E2E. Maybe @junjiezhang1997 can chime in to see how we can achieve that, perhaps in a separate PR?

Reply:
This variable is not used to deploy test vmss now. Actually, we obtain the latest image to deploy test vmss in https://github.com/Azure/AgentBaker/blob/master/e2e/windows/e2e-scenario.sh#L143.

Reply:
The variable windowsNodeImageVersion is not used to deploy test vmss now. We use https://github.com/Azure/AgentBaker/blob/master/e2e/windows/e2e-scenario.sh#L143 to obtain the latest image to deploy test vmss. So I think gpu case should already use latest image now. We can check it in Azure Portal when running the test.

Reply:
I just verified with @junjiezhang1997 that the test vm is indeed using the latest image.

Reply:
This variable is not used by test vm anymore. Create a task to remove it later.